README.md

```diff
@@ -1,37 +1,132 @@
-# CNCF Project Template Repository
+<!-- template begins here-->
 
-This is a [template repository][template-repo] for CNCF projects created by [CNCF SIG Contributor
-Strategy][contrib-strat]. You can use it to either start a new repository that
-has all the required files for a CNCF project or just grab the particular files
-that you need.
+# Welcome to the CNCF Artifacts Working Group!
 
-## Steps
+<!-- Mission Statement -->
 
-1. Click **Use this template** and create a copy of this repository.
+In the distributed world of cloud-native computing different artifacts and
+packages are used to transport configuration and code for the many services and
+capabilities that comprise and support workloads and applications. As an
+example, today ArtifactHub advertises that it can crawl about 20 such artifact
+types [1]. These "cloud-native" bundles add to the previous proliferation of
+package and artifact types for software dependencies such as Maven and npm and
+for system packages like deb and rpm. This abundance of package types and
+formats adds complexity and risk for cloud application developers and
+architects trying to provision and run cloud-native workloads. Different tools
+and controllers must be carefully learned and installed to bundle, unbundle,
+verify and deploy each artifact type. This complexity slows product development
+by users, impedes innovation and collaboration amongst projects, and increases
+risks of insecure configuration. To reduce this complexity and facilitate
+collaboration and innovation, WG Artifacts will gather stakeholders from many
+CNCF and open source projects offering packaging, distribution and deployment
+mechanisms for bundles of configuration and code.
```

Comment:
Great leadup to the goal. Should we add another sentence that suggests our goal isn't to consolidate all the existing package managers, rather to continue investing in the existing, avoiding the need to create yet new package managers?

Reply:
Hopefully one doesn't need to add a new pkg mgr. But should the need arise, at least don't have to think about what kinds of metadata queries need to be supported. Documents coming out of this WG should provide said guidance. Furthermore, software coming out of the WG is general enough to be re-used/repurposed.